src/middleware/express.ts

```diff
@@ -35,15 +35,18 @@ type Middleware = ReturnType<typeof commonMiddleware.express.makeMiddleware>;
 
 export async function makeMiddleware(
   logger: winston.Logger,
-  transport: LoggingWinston
+  transport: LoggingWinston,
+  skipParentEntryForCloudRun?: boolean
```

Comment:
Is this new parameter fully optional? (That is, could this break existing code?)

Reply:
Yes Daniel, this new parameter is fully optional.